views/button/optimize-missing-sizes.php

```diff
@@ -0,0 +1,39 @@
+<?php
+defined( 'ABSPATH' ) || die( 'Cheatin’ uh?' );
+
+$html_atts = '';
+
+if ( empty( $data['atts'] ) ) {
+	$data['atts'] = [];
+}
+
+if ( ! isset( $data['atts']['class'] ) ) {
+	// Class used for JS.
+	$data['atts']['class'] = 'optimize-missing-sizes';
+}
+
+if ( ! isset( $data['atts']['data-processing-label'] ) ) {
+	// Used for JS.
+	$data['atts']['data-processing-label'] = __( 'Optimizing...', 'imagify' );
+}
+
+$html_atts = $this->build_attributes( $data['atts'] );
+?>
+
+<a href="<?php echo esc_url( $data['url'] ); ?>"<?php echo $html_atts; ?>>
```

Comment:
```suggestion
<a href="<?php echo esc_url( $data['url'] ); ?>" <?php echo $html_atts; ?>>
```

Reply:
This space character is not needed, the method `build_attributes()` adds one before each attribute.
The method will be commited later but it will be a duplicate of [this one](https://github.com/wp-media/imagify-plugin/blob/e914395a5b582c7b4e1798eee7bd08d088e67d1b/inc/classes/class-imagify-settings.php#L661).